install/aws/dev/docker/nginx/nginx.conf

```diff
@@ -141,4 +141,17 @@
                  # Fallback route
                  rewrite ^\/(?!wikijump--next\/)(.*)$ /index.php?Wiki__WikiScreen/wiki_page/$1? break;
          }
+
+         # Health Checking for Datadog
+         server {
+               listen 81;
+               server_name localhost;
+               access_log off;
+               allow 172.16.0.0/12;
+               deny all;
```

Comment:
Does Datadog have any IPv6 addresses? (Assuming that's what this subnet is for)

Reply:
That's actually Docker's internal address space, this essentially restricts access to this page to containers.